pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -98,9 +103,45 @@ func New(context context.Context,
 	}
 	log.Info(sync.Device.Address, " Encoding:", sync.encoding, " Capabilities ", capResponse)
 
+	log.Infof("Getting initial configuration for device %s with type %s and version %s", device.ID, device.Type, device.Version)
+	onosExistingConfig, errConfigGet := devicechangeutils.ExtractFullConfig(devicetype.NewVersionedID(devicetype.ID(device.ID),
+		devicetype.Version(device.Version)), nil, deviceChangeStore, 0)
+	if errConfigGet != nil {
+		//TODO what should we do in this case
+		log.Errorf("Can't extract initial configuration for %s due to: %s", sync.Device.Address, errConfigGet)
+		//return nil, errConfigGet
+	}
+
+	if len(onosExistingConfig) != 0 {
+		errSet := initialSet(context, onosExistingConfig, device, target)
+		if errSet != nil {
+			//TODO what should we do in this case ?
+			log.Errorf("Can't set initial configuration for %s due to: %s", sync.Device.Address, errSet)
+			return nil, errSet
+		}
+	} else {
+		log.Infof("No pre-existing configuration for %s", device.ID)
+	}
 	return sync, nil
 }
 
+func initialSet(context context.Context, onosExistingConfig []*devicechange.PathValue, device *topodevice.Device, target southbound.TargetIf) error {
+	setRequest, errExtract := values.PathValuesToGnmiChange(onosExistingConfig)
+	log.Info("Setting initial configuration for device %s : %s", device.ID, setRequest)
+	if errExtract != nil {
+		return errExtract
+	}
+	setResponse, errSet := target.Set(context, setRequest)
```

Comment:
Here I think you're just going straight to the Target - not to SetConfig in the manager - no validation will be done. 
Maybe just send this to log for the moment until next stage